dist/stat_data.py

```diff
@@ -285,6 +287,10 @@ def __init__(self, name, desc, flags=''):
     TxnStat('txn_snapshots_created', 'number of named snapshots created'),
     TxnStat('txn_snapshots_dropped', 'number of named snapshots dropped'),
     TxnStat('txn_sync', 'transaction sync calls'),
+    TxnStat('txn_fsync_pre', 'transaction fsync calls before creating a transaction id'),
```

Comment:
These should be added in the list in alphabetical order. I'm also inclined to rename them so they sort alphabetically how you want, i.e change `txn_fsync_duration_pre` to be `txn_fsync_pre_duration` similarly for post.

Minor phrasing change, I'd like to switch from:
``calls before creating a transaction id``
To:
``calls for checkpoint before allocating the transaction ID``

Similarly for the other descriptions as well.